<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectDialog</class>
 <widget class="QDialog" name="ConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
     <item>
      <widget class="QLabel" name="LabelUserName">
       <property name="text">
        <string>username</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUserName">
       <property name="placeholderText">
        <string>username</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
     <item>
      <widget class="QLabel" name="labelIp">
       <property name="text">
        <string>IP</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditIp">
       <property name="text">
        <string>127.0.0.1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
     <item>
      <widget class="QLabel" name="labelPort">
       <property name="text">
        <string>port</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPort">
       <property name="text">
        <string>10086</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QComboBox" name="comboBoxColor">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BLACK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>WHITE</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOutLine">
       <property name="text">
        <string>OutLine</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>onClickConnectSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOutLine</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>onClickOutlineSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>236</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExit</sender>
   <signal>clicked()</signal>
   <receiver>ConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClickConnectSlot()</slot>
  <slot>onClickOutlineSlot()</slot>
 </slots>
</ui>
